index.d.ts

```diff
@@ -521,7 +521,7 @@ declare namespace Moleculer {
 		disconnected(): void;
 	}
 
-	class Context<P = unknown, M extends object = {}> {
+	class Context<P = unknown, M extends object = {}, L extends object = GenericObject> {
```

Comment:
I don't believe there are any constraints to what type `ctx.locals` can be.  According to the docs, this is of `any` type.  I don't think this `extends` is necessary.

In inspecting the moleculer source, it appears that this property is initialized to an empty object.  I'd lean towards the default value of this type being `Record<string, never>` since that's the actual type in place unless the consumer makes changes to it.  So, I think this should be `L = Record<string, never>`.

Reply:
I think `L = GenericObject` will better, It will keep old code work without changes

![Screen Shot 2022-03-31 at 03 07 10](https://user-images.githubusercontent.com/32500015/160921811-c75a3945-f21a-4b0a-8383-4b9b32bbb7b7.png)